pmlc/dialect/tile/transforms/contraction.cc

```diff
@@ -209,6 +209,10 @@ static IndexAccess ConvertAffineMap(ContractionOp op, mlir::AffineMap map) {
 }
 
 Contraction::Contraction(ContractionOp op) {
+  if (VLOG_IS_ON(2)) {
+    auto op_result = op.result();
+    IVLOG(2, "Processing: " << mlir::debugString(op_result));
+  }
```

Comment:
`op.result()` is basically a zero-cost thing to do, so no need to wrap this in `VLOG_IS_ON(2)`